karma.conf.js

```diff
@@ -0,0 +1,72 @@
+// Karma configuration
```

Comment:
A bit nitty, but you could copy it from my marketplace-gulp PR, I cleaned up the comments, 80-char stuff, etc.